patches/minecraft/net/minecraft/network/datasync/EntityDataManager.java.patch

```diff
@@ -5,7 +5,7 @@
  
     public static <T> DataParameter<T> func_187226_a(Class<? extends Entity> p_187226_0_, IDataSerializer<T> p_187226_1_) {
 -      if (field_190303_a.isDebugEnabled()) {
-+      if (true || field_190303_a.isDebugEnabled()) { // Forge: This is very useful for mods that register keys on classes that are not their own
++      if (true) { // Forge: This is very useful for mods that register keys on classes that are not their own
```

Comment:
Why are you making this change? It serves no real purpose.

Reply:
I figured there was no point to having the `field_190303_a.isDebugEnabled()` part, it's a little confusing to leave it there since it doesn't affect the logic.